### DISCUSSION TITLE
Springboot项目引入依赖后无法使用相关内容

### DISCUSSION THREAD

**Discussion by xiaodi-hub** [ForteScarlet/simpler-robot#444] [不要在这里发帖子了]
[](https://qiniuyun.xdskr.cn/image-20220907164607366.png)
项目是Springboot项目 依赖是：
        <dependency>
            <groupId>love.forte.simple-robot</groupId>
            <artifactId>component-mirai-spring-boot-starter</artifactId>
            <version>2.3.8</version>
        </dependency>
不知道有人跟我遇到过一样的情况吗？

**Comment by ForteScarlet** [comment]
参考 [常见问题-导包异常](https://www.yuque.com/simpler-robot/simpler-robot-doc/rvsl9p)
